CMakeLists.txt

```diff
@@ -199,12 +200,12 @@ install(
   DIRECTORY ${PROJECT_SOURCE_DIR}/amr-wind
   DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}
   FILES_MATCHING PATTERN "*.H")
-
+  
 install(
   EXPORT ${PROJECT_NAME}Targets
   NAMESPACE ${PROJECT_NAME}::
   DESTINATION lib/cmake/${PROJECT_NAME})
-
+  
```

Comment:
Can you remove these accidental spaces?

Reply:
@jrood-nrel I have removed the spaces here - https://github.com/Exawind/amr-wind/pull/551/commits/6d335317b88b0152e4b1e970a886226d4e92b1fa